pkg/types/types.go

```diff
@@ -57,9 +57,11 @@ type PerformLogProvider interface {
 }
 
 type PerformLog struct {
-	Key           UpkeepKey
-	TransmitBlock BlockKey
-	Confirmations int64
+	Key             UpkeepKey
+	TransmitBlock   BlockKey
+	Confirmations   int64
+	TransactionHash string
+	BlockNumber     int64
```

Comment:
We also need to ensure these get populated.  @RyanRHall @EasterTheBunny 

Also is blockNumber the same as transmit block? do we need both?

Reply:
blockNumber should indicate the blocknumber that includes the tx which produced the logs - different from transmit block

Reply:
I think that is the transmit block number. As far as i understand there is checkBlockNumber: block at which check was done, transmitBlockNumber: block at which transmit was confirmed. cc @EasterTheBunny to correct me